doxygen/Doxyfile

```diff
@@ -2176,7 +2176,7 @@ INCLUDE_FILE_PATTERNS  =
 # recursively expanded use the := operator instead of the = operator.
 # This tag requires that the tag ENABLE_PREPROCESSING is set to YES.
 
-PREDEFINED             =
+PREDEFINED             = RMM_NAMESPACE
```

Comment:
Sorry, I updated after the fact, this needs to go in `EXPAND_AS_DEFINED`, and we should put `RMM_EXPORT` in this list.